src/main/static/src/components/Raamit.js

```diff
@@ -174,10 +172,7 @@ export default class Raamit extends React.Component {
                 });
             }
 
-            // Map all item's properties to array's objects
-            return mapKeys(item, (key, value) => {
-                return value;
-            });
```

Comment:
En tiedä mitä tässä oli yritetty tehdä, mutta taulukoksi tämä ei ainakaan tällä muutu. Tässä mäpättin avaimet avaimiksi (funktion parametrien nimet väärin päin), mikä ei käytännössä muuta objektia mitenkään. https://lodash.com/docs/4.17.15#mapKeys